### PR TITLE
feat: edit wallet

### LINKED ISSUE #121: edit wallet & account

### PR DESCRIPTION
**Pull request by Johnson-OpNet**
Edit wallet and account, close #121

### CODE REVIEW

File: app/src/main/java/com/cj/bunnywallet/datasource/local/WalletDataStore.kt [hunk 1]
```diff
@@ -72,7 +87,37 @@ class WalletDataStore @Inject constructor(private val dataStore: DataStore<Walle
...
-            Timber.d(message = "Failed to update wallet preferences: ${e.message}")
+            Timber.d(message = "Current account failed: ${e.message}")
+        }
+    }
+
+    suspend fun editWalletName(walletId: String, newName: String) {
+        try {
+            dataStore.updateData {
+                it.copy {
+                    val w = wallets[walletId] ?: throw Throwable("Not find wallet")
```
Comment (wannnn) [inline app/src/main/java/com/cj/bunnywallet/datasource/local/WalletDataStore.kt]: typo "Not found "

File: app/src/main/java/com/cj/bunnywallet/datasource/local/WalletDataStore.kt [hunk 2]
```diff
@@ -72,7 +87,37 @@ class WalletDataStore @Inject constructor(private val dataStore: DataStore<Walle
...
+            Timber.d(message = "Edit wallet name failed: ${e.message}")
+        }
+    }
+
+    suspend fun editAccountName(walletId: String, address: String, newName: String) {
+        try {
+            dataStore.updateData {
+                it.copy {
+                    val w = wallets[walletId] ?: throw Throwable("Not find wallet")
+                    val acc = w.accountsMap[address] ?: throw Throwable("Not find account")
```
Comment (wannnn) [inline app/src/main/java/com/cj/bunnywallet/datasource/local/WalletDataStore.kt]: Not found

File: app/src/main/java/com/cj/bunnywallet/datasource/local/WalletDataStore.kt [hunk 3]
```diff
@@ -41,25 +41,40 @@ class WalletDataStore @Inject constructor(private val dataStore: DataStore<Walle
...
+            Timber.d(message = "Create wallet failed: ${e.message}")
         }
     }
 
-    suspend fun createAccount(walletId: String, acc: Account) {
+    suspend fun deleteWallet(walletId: String) {
         try {
             dataStore.updateData {
                 it.copy {
-                    val w = wallets[walletId] ?: throw Throwable("Not find wallet")
```
Comment (wannnn) [inline app/src/main/java/com/cj/bunnywallet/datasource/local/WalletDataStore.kt:52]: Not found

Comment (Johnson-OpNet) [reply app/src/main/java/com/cj/bunnywallet/datasource/local/WalletDataStore.kt:52]: This is a delete line..
Que pasa ?

**Review by wannnn — APPROVED** [review]

**Pull request merged**